README.md

```diff
@@ -0,0 +1,73 @@
+# Sunshine Plays Tetris®!
+
+This is a chat-powered multiplayer cooperative Tetris game made possible by [Zendesk Sunshine Conversations](https://smooch.io).
+
+## Prerequisites
+
+### Zendesk Sunshine Conversations Account
+
+- Create a [Sunshine account](https://smooch.io/signup/) if you don't already have one, or [log into it](https://app.smooch.io/login).
+- Create a new app (you may follow along [this documentation](https://docs.smooch.io/guide/api-quickstart/)).
+- In your new app, go to the `Settings` tab and create new [API key](https://docs.smooch.io/guide/authentication-overview/#api-keys).
+
+### Environment Variables
+
+In the root of your project, make a copy of the `.env.example` file and name it `.env` (`cp .env.example .env`)
```

Comment:
Should gitignore that file.

Reply:
The `.env` file? It already is https://github.com/zendesk/sunshine_plays_tetris/blob/d73531becff58071dd2689928b1aebfccff96fc2/.gitignore#L2

Reply:
À quoi j'pensais? J'avais le feeling que t'avais créé le truc de config mais onon t'as raison c'était deja la.